lib/xilinx/frames.cc

```diff
@@ -35,65 +38,67 @@ int Frames::readFrames(const std::string& frm_file_str) {
 
 		std::vector<std::string> frame_data_strings =
 		    absl::StrSplit(frame_delta.second, ',');
-		if (frame_data_strings.size() != 101) {
```

Comment:
This block still seems important?  Why was it removed?

Reply:
This condition needs specialization when introducing architectures where data in frames can have a variable number of words, e.g. Spartan6. But here this condition still stands and should be reinstated.